modules/synacormediaBidAdapter.js

```diff
@@ -85,6 +85,17 @@ export const spec = {
       }
     });
 
+    // CCPA
+    if (bidderRequest && bidderRequest.uspConsent) {
+      if (!openRtbBidRequest.regs) {
+        openRtbBidRequest.regs = {};
+      }
+      if (!openRtbBidRequest.regs.ext) {
+        openRtbBidRequest.regs.ext = {};
+      }
+      openRtbBidRequest.regs.ext.us_privacy = bidderRequest.uspConsent;
```

Comment:
I would suggest using `utils.deepSetValue()` method instead of all the if statements here (you'll need to import `utils` as well)

```suggestion
      utils.deepSetValue(openRtbBidRequest, 'regs.ext.us_privacy', bidderRequest.uspConsent);
```